chunky/src/java/se/llbit/chunky/main/Chunky.java

```diff
@@ -488,6 +489,7 @@ public List<ContextMenuTransformer> getMapContextMenuTransformers() {
 
   /**
    * Get the mutable list of context menu transformers in the main render view.
+   * The canvas X and Y coordinates are supplied as a `Vector2` in the context menu's `canvasPosition` property.
```

Comment:
Also, could you please clarify the origin for the coordinate system (which corner) and if it is for the surrounding FX element or the canvas itself (and how does scaling affect these? Do you get the pixel location for lookup in the render buffer or something on which you have conversion to do?).

Reply:
It's the coordinate of the canvas which is the same as the pixel location on the render buffer. I updated the javadocs.